js/start.md

```diff
@@ -275,454 +229,570 @@ To install React-specific Amplify UI components, run the following command:
+$ npm install --save @aws-amplify/api @aws-amplify/pubsub
+$ npm start
 ```
 
-In addition to *aws-amplify* core, you can install our angular module which provides a service provider, helpers, and components:
+In addition to `aws-amplify` core, you can install the Angular Ionic modules which provide a service provider, helpers, and components:
 
 ```bash
 $ npm install --save aws-amplify-angular
 ``` 
 
-See the [Angular Guide](https://aws-amplify.github.io/amplify-js/media/angular_guide){: target='_new'} for details and usage.
+See the [Ionic Guide](https://aws-amplify.github.io/docs/js/angular){: target='_new'} for details and usage.
 {: .callout .callout--action}
 
 </div>
-<div id="ionic" class="tab-content">
 
-```bash
-$ npm install --save aws-amplify
-```
+<div id="vue" class="tab-content">
 
-In addition to `aws-amplify` core, you can install our angular and ionic modules which provide a service provider, helpers, and components:
-
-</div>
-<div id="ionic" class="tab-content">
+Use the Vue CLI to bootstrap a new Vue app (selecting the defaults will work for this project):
 
 ```bash
-$ npm install --save aws-amplify-angular
-``` 
-
-See the [Ionic Guide](https://aws-amplify.github.io/amplify-js/media/ionic_guide){: target='_new'} for details and usage.
-{: .callout .callout--action}
+$ npm install -g @vue/cli
+$ vue create myAmplifyProject 
+$ cd myAmplifyProject
+```
 
-</div>
-
-<div id="vue" class="tab-content">
+Inside the app directory, install Amplify and run your app:
 
 ```bash
-$ npm install --save aws-amplify
+$ yarn add @aws-amplify/api @aws-amplify/pubsub
+$ yarn serve
 ```
 
 To install Vue-specific Amplify UI components and the Amplify Vue plugin, run the following command:
 
 ```bash
-$ npm install --save aws-amplify-vue
+$ yarn add aws-amplify-vue
 ```
 
 See the [Vue Guide](https://aws-amplify.github.io/docs/js/vue){: target='_new'} for details and usage.
 {: .callout .callout--action}
-</div>
 
 </div>
 
-## Step 3. Set up the AWS Backend
+## Step 2: Set Up Your Backend
 
-<div class="nav-tab setup" data-group='setup'>
-<ul class="tabs">
-    <li class="tab-link purejs current" data-tab="purejs">JavaScript</li>
-    <li class="tab-link react" data-tab="react">React</li>
-    <li class="tab-link react-native" data-tab="react-native">React Native</li>
-    <li class="tab-link angular" data-tab="angular">Angular</li>
-    <li class="tab-link ionic" data-tab="ionic">Ionic</li>
-    <li class="tab-link vue" data-tab="vue">Vue</li>
-</ul>
-
-Create new AWS backend resources and pull the AWS services configuration into the app. In a terminal window, change to the root directory of your app and run the following command (for this app, accepting all defaults is OK).  When asked for the name of an environment, enter 'dev'.
+In a terminal window, navigate to your project folder and run the following command (accept defaults is OK, use 'test' environment name):
 
 ```bash
-$ amplify init
+$ cd ./YOUR_PROJECT_FOLDER
```

Comment:
I believe you  already had the reader cd into the project directory (i.e. 'cd myAmplifyProject')